integration/tests/cook/mesos.py

```diff
@@ -0,0 +1,122 @@
+import logging
+import os
+import sys
+from urllib.parse import urlparse, parse_qs
```

Comment:
Nitpick: Can we import `urllib.parse` as `up` and then qualify the accesses to these methods with `up.` below?

Reply:
Done